proc-macros/src/helpers.rs

```diff
@@ -165,6 +165,18 @@ pub(crate) fn is_option(ty: &syn::Type) -> bool {
 	false
 }
 
+/// Iterates over all Attribute's and parses only the attributes that are doc comments.
+///
+/// Note that `doc comments` are expanded into `#[doc = "some comment"]`
+/// Thus, if the attribute starts with `doc` => it's regarded as a doc comment.
+pub(crate) fn extract_doc_comments(attrs: &[syn::Attribute]) -> TokenStream2 {
+	let docs = attrs.iter().filter(|attr| attr.path.is_ident("doc")).filter(|attr| match attr.parse_meta() {
```

Comment:
I'm not super happy with the formatting of this one but it's better then previous version at least.....

Reply:
Could have been `.filter(|attr| attr.path.is_ident("doc") && match ...` :see_no_evil:

Reply:
:cold_face: I can fix in an follow up PR